subsystems/provisioning/setup.go

```diff
@@ -139,3 +141,24 @@ func (w *Provisioning) initDevices() error {
 
 	return nil
 }
+
+func (w *Provisioning) enableWifi(ctx context.Context) error {
+	if err := w.nm.SetPropertyWirelessEnabled(true); err != nil {
+		return err
+	}
+
+	timeoutCtx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+	for {
```

Comment:
Without this loop the first scan fails and throws an error, as it takes a couple seconds for the radio to power on.